.github/workflows/main.yml

```diff
@@ -269,13 +269,12 @@ jobs:
         node-version: [10.x, 14.x]
     steps:
       - uses: actions/checkout@v1
-      - name: Use Node.js ${{ matrix.node-version }}
       - uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
       - name: Install dependencies w/o lockfile
         run: yarn install --no-lockfile --non-interactive
-      - name: Basic Tests
+      - name: Basic Tests with Node.js ${{ matrix.node-version }}
```

Comment:
You want to add the `name` to the job, not the step

Reply:
like this https://github.com/ember-cli/ember-cli/blob/21536af8414e70a3e66ab3ae0d64cce24b309f0b/.github/workflows/ci.yml#L49

Reply:
Oh interesting. Looks like it isn't a pattern we use but certainly a pattern we can start using here.  Fixed.